rds-deployment/azuredeploy.json

```diff
@@ -108,7 +108,7 @@
     "lbBE": "ADLBBE",
     "publicBEAddressPoolID": "[concat(variables('publiclbID'),'/backendAddressPools/',variables('lbBE'))]",
     "gwLBName": "[concat('GWPLB',resourceGroup().name)]",
-    "publicIPAddressName": "[tolower(concat('adpip',resourceGroup().name))]",
+    "publicIPAddressName": "[tolower(concat('adpip',uniqueString(resourceGroup().Id)))]",
```

Comment:
This adds some unnecessary complexity since the PIP name is not globally unique.  A simple literal string will work here.